MyLibrary/BusinessLogicTests/AuthorServiceTests.cs

```diff
@@ -0,0 +1,207 @@
+// <copyright file="AuthorServiceTests.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using BusinessLogic.DataProvider;
+using BusinessLogic.LibraryModel;
+using BusinessLogic.LibraryService;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace BusinessLogicTests
+{
+    /// <summary>
+    /// Tests for Author Services
+    /// </summary>
+    [TestClass]
+    public class AuthorServiceTests
+    {
+        /// <summary>
+        /// Tests the method GetAuthorById
+        /// Correct input values
+        /// Returned correct author
+        /// </summary>
+        /// <param name="id">The author identifier</param>
+        /// <param name="name">The author name</param>
+        [TestMethod]
+        [DataRow(1, "TestAuthor")]
+        public void GetAuthorById_CorrectId_ReturnedCorrectAuthor(int id, string name)
+        {
+            // Arrange
+            Author expectedAuthor = new Author { Id = id, FullName = name };
+            var mockDataProvider = new Mock<IDataProvider>();
```

Comment:
You use such declaration of mock in every test method, such things good to extract to '[Test Initialize]' method, declared the private variable in the test class. I did so.

Reply:
No, we need to set up Mock in every test, not for all tests at once